### PR TITLE
Feature/phrase board

### PR DESCRIPTION
**Pull request by BraydenRoyston**
added phrase board
- early version of guessing (user types out entire sentence, when they hit the enter button it'll get thrown up into the respective guess row)
- we need validation on the enter keystroke to ensure that the number of characters in the guessed phrase is correct
- eventually we want to cast the letters that the user types onto the letter boxes themselves instead of showing the guess in the box above the keyboard, will work on this after we figure out valiidation

### CODE REVIEW

File: phrasle/src/components/word/phrase/phrase.jsx
Comment (BraydenRoyston) [inline phrasle/src/components/word/phrase/phrase.jsx:45]: The following is highly disgusting and I want suggestions

**Comment by ericdachen** [comment]
LGTM, I'm gonna branch off this once its merged

**Review by ericdachen — APPROVED** [review]

**Pull request merged**
